### DISCUSSION TITLE
RightToAsk meetup

### DISCUSSION THREAD

**Discussion by KipCrossing** [RightToAskOrg/righttoask-docs#9] [Q&A]
Would there be interest in a RightToAsk meetup (online) to discuss the scope/vision of the community and its related projects?

**Comment by vteague** [comment]
@KipCrossing we have a weekly meeting at 2.15 Mondays (AEST) and an online chat server, which you're very welcome to join, as is anyone else who is interested - I'll send you an invitation.

These tend to be focused on the technical questions of the week (at the moment we're obsessing about UI design and the server API) so there may be a good role for a broader meetup among people who aren't actively working on the project, but are interested in suggesting ideas and discussing directions. What do you think?

(Anyone else reading this who would also like to join, just ask.)
